content/wasm-functions/using-cron-jobs.md

```diff
@@ -0,0 +1,227 @@
+        }
+    });
+
+// Attach the router to the fetch event
+addEventListener('fetch', (event) => {
+    event.respondWith(router.fetch(event.request));
+});
+```
+
+To build and run the application locally, run the following command:
+
+<!-- @selectiveCpy -->
+
+```console
+spin build --up
+```
+
+Let's curl the application's endpoint to test if it is working as expected:
+
+<!-- @selectiveCpy -->
+
+```console
+curl localhost:3000/cron-me
+```
+
+ You should see a message returned along the lines of (_but with an accurate datetime at the time of following this tutoral_):
+
+<!-- @nocpy -->
+
+```console
+Hello! This is a direct ping at 2025-04-02T02:45:25.562Z
+```
+
+## Deploy Spin Application to Fermyon Wasm Functions
+
+Before scheduling our cron job, we deploy our Spin application to the Fermyon Wasm Functions platform. Deploy the application with the following command:
+
+<!-- @selectiveCpy -->
+
+```console
+spin aka deploy
+```
+
+Upon successful deployment, you should receive a domain name that you can use to test the application:
+
+<!-- @nocpy -->
+
+```console
+curl https://ec8f3c23-b73d-4d51-aa69-aabd7ddc132a.aka.fermyon.tech/cron-me
+Hello! This is a direct ping at 2025-04-02T03:16:14.212Z% 
+```
+
+# Scheduling a Cron Job for Your Spin Application
+
+Now we will use the `spin aka crons` command to invoke the Spin app's HTTP endpoint on our desired schedule. The syntax for this request follows standard crontab format.
+
+> `spin aka crons` supports multiple digits in each cron expression field and intervals in every position (e.g., `* */12 * * *` for every 12 hours). Syntax elements like comma-separated lists and ranges are currently supported. Scheduling follows UTC and you may need to [convert your local time to UTC](https://www.worldtimebuddy.com/?pl=1&lid=100&h=100&hf=1) to ensure correct execution during daylight savings time.
+
+Let’s create a cron job that triggers your Spin application every 5 minutes. The `spin aka crons create` subcommand takes three arguments:
+
+* **Schedule:** How often the job runs, using standard cron syntax
+* **Path and query:** The HTTP path (and optional query parameters) to invoke
+* **Name:** The Spin applciaton to apply the cron job to
+
+In this example, we’ll schedule a job that hits the `/cron-me` path with a `msg` query parameter of "fwf" every 5 minutes for a Spin app named hello-cron-job.
+
+<!-- @selectiveCpy -->
+
+```console
+spin aka crons create "*/5 * * * *" "/cron-me?msg=fwf" "hello-cron-job"
+```
+
+You should see output similar to:
+
+<!-- @nocpy -->
+
+```console
+Created cron which will next run at 2025-04-02 04:00:00 UTC
+```
+
+If we wait 5 minutes and check our application logs with the `spin aka logs` subcommand, we will see that our cron job invoked the application at the top of the hour as expected. 
+
+<!-- @nocpy -->
+
+```console
+spin aka logs -a hello-cron-job
+2025-04-02 04:00:00 [hello-cron-job]  Hello, cron job triggered with msg: "fwf" at 2025-04-12T04:00:00Z
```

Comment:
Consider linking this back to the `console.log` statement from from the HTTP application, and maybe put some more motivation around this, both here and when you write the application.  E.g. motivating the application a la "For a real cron job, you'd see its effects in the data it updates or the artifacts it produces. For this tutorial, though, we'll just emit a log, so that we can see the application has been called." and then down here we can say "This is the message we had the application log so that we could see it was being called" or something like that.

(If you keep the "different behaviour when they try it interactively" thing then it would be good to be _extra_ clear about this because the message will be unfamiliar.)

Reply:
Great suggestion! I added a disclaimer above the source code snippet that this is a simplified example and allude to what operations you may see in a real world example.